k8s101.md

```diff
@@ -74,17 +76,18 @@ Events:
   32m		32m		1	{kubelet 172.28.128.5}	spec.containers{my-nginx}	Normal		Started		Started container with docker id f322f4208102
 ```
 
-Let's see what's inside the pod.
+Now let's focus on what's inside one pod.
 
-**Pod IPs**
+### Pod IPs
 
-You can spot IP in the overlay network assigned to pod. In my case it's `10.244.33.109`. Can we access it directly?
+In the Pods description output you should be able to spot the field `IP` in the overlay network assigned to the pod.
+In the example above it's `10.244.33.109`. Can we access it by using that IP directly?
 
-Let's try and see!
+Let's temporarily (`--rm`) run a Pod providing `curl` to verify if we can access the `nginx` Pod from other pods:
 
-```
-kubectl run -i -t --rm cli --image=tutum/curl --restart=Never
-curl http://10.244.33.109
+```bash
+$ kubectl run -i -t --rm cli --image=appropriate/curl --restart=Never
```

Comment:
You need to override entrypoint for the `appropriate/curl` image to put you into shell.

```bash
kubectl run -ti --rm cli --image=appropriate/curl --restart=Never --command /bin/sh
```